host_test.go

```diff
@@ -107,9 +117,7 @@ func TestHostNameCollision(t *testing.T) {
 		scanner := bufio.NewScanner(r)
 
 		// Consume the initial buffer
-		scanner.Scan()
```

Comment:
I think there are only one case where N>5 (:> N=8) - do you think it is worth keeping the function then? The eight calls to `scanner.Scan()` aren't "pretty" but at least there is one way to do it (decrease the cognitive cost of figuring out what is going on). Wdyt?